core/src/test/kotlin/io/github/serpro69/kfaker/RandomServiceTest.kt

```diff
@@ -305,6 +324,19 @@ internal class RandomServiceTest : DescribeSpec({
                 }
             }
         }
+
+        context("nexDate(min, max, zoneOffset) fun") {
+            val min = Instant.ofEpochSecond(0)
```

Comment:
I reduced the range between `min` and `max`.
I also reviewed the `randomPastDate` and `randomFutureDate` behavior to avoid the possibility to have inclusive value.

- Past: range from `Unix epoch` to `Instant.now()` exclusive
- Future: range from `Instant.now()` exclusive to `Instant.now() + 50years`

Reply:
Awesome!
How about making the `randomFutureDate` take an argument `years` with  `50` as default value? To provide the possibility to supply your own number of years as well.

Reply:
Yes good idea, we can have the `randomFutureDate()` with the default value and `randomFutureDate(Duration)` for exemple.

Reply:
Done :)